entity-types/mobile-application/golden_metrics.yml

```diff
@@ -17,22 +16,19 @@ httpResponseTimeMsMetric:
   title: HTTP response time (ms)
   unit: MS
   query:
-    select: average(newrelic.timeslice.value) * 1000
-    where: metricTimesliceName = 'External/all'
+    select: average(apm.mobile.external.duration) * 1000
```

Comment:
This metric will return the same average values as the old `External/all` metric, but `apm.mobile.external.duration` is actually mapped to `External/{host}/all`.  As a result, we will process more values in order to calculate the total.  That shouldn't be an issue.  On the alerting side, when customers create an alert with this metric, they will have the option of faceting on the host, which is a nice usability improvement.